pt.html

```diff
@@ -161,36 +161,42 @@
 <panel w=500 h=250>
 <pic src="pics/intro0.png" sx=400 sy=250></pic>
 <words x=155 y=20 w=90 no-bg>
-	Yeah.
+	É assim.
 </words>
 <words x=196 y=65 w=270 no-bg>
-	Not only are common practices like lectures, cramming, and re-reading <i>boring</i>,
-	science has shown they <i>don't even work well</i>.*
+	Não apenas as práticas comuns, como palestras, <i><abbr title="Ato de decorar quantidades massivas no dia anterior de um teste" lang="en">cramming</abbr></i>
+	e releituras são <i>chatas</i>,
+	a ciência tem mostrado que <i>nem mesmo funcionam bem</i>.*
+	<!--
+	Literal: 'Não apenas as práticas comuns, como palestras, cramming e releituras chatas, a ciência tem mostrado que elas nem funcionam bem.'
+	Term: 'cramming'
+		Wikipedia: In education, cramming (also known as mugging or swotting, from swot, akin to "sweat", meaning "to study with determination") is the practice of working intensively to absorb large volumes of informational material in short amounts of time.
+		urbandictionary.com: v; To study the day of. Studying during homeroom or the preceding periods of the test.
+	-->
 </words>
 </panel>
 
 <panel w=500 h=30 style="margin-top:-5px">
 <words w=500 x=-15 y=-15 no-bg style="width: 500px; font-size:0.8em; text-align: right; color:#999;">
-	* all sources and links will be at the end of this comic!
+	* todas as fontes e links estarão no final desta história em quadrinhos!
 </words>
 </panel>
 
 <panel w=500 h=350>
 <pic src="pics/intro0.png" sx=400 sy=500></pic>
 <words x=95 y=30 w=340 no-bg>
-	But what if I said there's a way to learn
-	that's evidence-based <i>and</i> fun?
+	Mas e se eu disser que há uma maneira de aprender que é baseada em evidências <i>e</i> divertida?
```

Comment:
Essa não encaixa no balão.

Só pensei em mudar para:

`Mas e se eu disser que há uma maneira que funciona?`

Reply:
Por enquanto, acho que vale a pena a gente se focar mais na tradução dos textos para português, e depois (se reeeealmente for necessário) alterar frases para forçar se ajustar com o que está no original.

Porém todos as alterações com traduções em si são extremamente interessantes